pepper-apis/docs/specification/src/endpoints/studies.invitation-check.yml

```diff
@@ -0,0 +1,35 @@
+post:
+  operationId: checkInvitationStatus
+  tags:
+    - Invitations
+  summary: CheckInvitationStatus
+  description: Check the status of an invitation, whether it exits and is valid or not.
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          required:
+            - auth0ClientId
+            - auth0Domain
+            - invitationId
+            - recaptchaToken
+          properties:
+            auth0ClientId:
+              description: the id of the client
+              type: string
+            auth0Domain:
+              description: the domain of the client
```

Comment:
Why not just the study guid?

Reply:
Study guid is already in the URL path. Requiring auth0 client/domain here so we know which client is calling in, and hopefully acts as a bit more deterrent for whackos.